src/openagi/actions/tools/github_tool.py

```diff
@@ -0,0 +1,75 @@
+import os
+from github import Github
+from openagi.actions.base import BaseAction
+from pydantic import Field
+from typing import Optional
+from github.GithubException import GithubException
+from dotenv import load_dotenv
+load_dotenv()
+
+class GithubTool(BaseAction):
+    """Create file Action"""
+
+    query: str = Field(..., description="action to be performed on github")
+    access_token: str = Field(default=os.environ['GITHUB_ACCESS_TOKEN'], description="GitHub Personal Access Token.")
```

Comment:
access these values from env inside `execute()` method. Adding here might include the secrets in the prompt.